lib/utils/report.mjs

```diff
@@ -142,8 +142,6 @@ function printfLike(format, ...args) {
  * @param {DisabledRangeObject} disabledRanges
  */
 function isDisabled(ruleName, startLine, disabledRanges) {
-	if (!disabledRanges) return false;
```

Comment:
note

https://github.com/stylelint/stylelint/pull/380/files#diff-8d2b4819892fef9fe283797c5d2a4ef3ccd4aac43b1e25da50f5db420f5bbb27R31

Reply:
`disabledRanges` should always be truthy, thanks to #7728.